### PR TITLE
Update ElevenToon.mjs

### PR DESCRIPTION
**Pull request by Robonau**
changed url to current and added _getMangaFromURI

### CODE REVIEW

File: src/web/mjs/connectors/ElevenToon.mjs [hunk 1]
```diff
@@ -13,7 +14,14 @@ export default class ElevenToon extends Connector {
         super.id = '11toon';
         super.label = '11toon';
         this.tags = [ 'manga', 'korean' ];
-        this.url = 'http://www.11toon.com';
```
Comment (27136773) [inline src/web/mjs/connectors/ElevenToon.mjs:16]: May add same mechanism as Heaventoon/Holymanga, when domain keeps being changed regulary
https://github.com/manga-download/hakuneko/blob/9d7264bd2e30809a897a17ddc677c78183bbbd8d/src/web/mjs/connectors/HolyManga.mjs#L13-L28

**Review by 27136773 — APPROVED** [review]
Looks good

**Comment by Robonau** [comment]
cant actually test any of this cause of the no javascript cloudflare thing

File: src/web/mjs/connectors/ElevenToon.mjs [hunk 2]
```diff
@@ -13,14 +13,28 @@ export default class ElevenToon extends Connector {
...
+        /*
+         * sometimes cloudflare bypass will fail, because chrome successfully loads the page from its cache
+         * => append random search parameter to avoid caching
+         */
+        let uri = new URL(this.url);
+        uri.searchParams.set('ts', Date.now());
+        uri.searchParams.set('rd', Math.random());
+        let request = new Request(uri.href, this.requestOptions);
+        let dat = await fetch(request);
+        this.url = new URL(dat.url).origin;
```
Comment (Robonau) [inline src/web/mjs/connectors/ElevenToon.mjs]: used fetch to get around the cloudflare issue
cause the response is still from the new url

File: src/web/mjs/connectors/ElevenToon.mjs [hunk 3]
```diff
@@ -1,127 +1,93 @@
...
-                        .then( chapters => chapterList.concat( chapters ) );
-                } else {
-                    return Promise.resolve( chapterList );
-                }
-            } );
+    async _getMangaFromURI(uri) {
+        const request = new Request(new URL(uri), this.requestOptions);
+        const data = await this.fetchDOM(request, '#cover-info h2');
+        const title = data[0].textContent.trim();
+        return new Manga(this, uri, title);
```
Comment (27136773) [inline src/web/mjs/connectors/ElevenToon.mjs]: Use `uri.pathname`, the manga id should not be an `URL` object

**Review by 27136773 — COMMENTED** [review]
Can be merged as soon as Lint error and manga id are adjusted

**Pull request merged**